### PR TITLE
fix retrieving user in lock srvice

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: models/classes/user/implementation/UserLocksService.php [hunk 1]
```diff
@@ -136,15 +136,18 @@ private function increaseLoginFails($login)
...
      */
     public function lockUser(User $user)
     {
-        $currentUser = UserHelper::getUser(tao_models_classes_UserService::singleton()->getCurrentUser());
+        $currentUser = tao_models_classes_UserService::singleton()->getCurrentUser();
 
         if (!$currentUser) {
             $currentUser = $user;
+        } else {
+            $currentUser = UserHelper::getUser(tao_models_classes_UserService::singleton()->getCurrentUser());
```
Comment (Alroniks) [inline models/classes/user/implementation/UserLocksService.php:150]: Seems `tao_models_classes_UserService::singleton()->getCurrentUser()` already assigned to `$currentUser`, so no need to fetch it again.  Solution would be `UserHelper::getUser($currentUser);`

But in this case, it makes sense to improve the whole condition.

File: models/classes/user/implementation/UserLocksService.php [hunk 2]
```diff
@@ -233,12 +236,15 @@ public function isLockable(User $user)
     public function getLockoutRemainingTime($login)
     {
         $lastFailure = $this->getLockout()->getLastFailureTime($login);
+        $result = false;
```
Comment (Alroniks) [inline models/classes/user/implementation/UserLocksService.php:239]: It is possible to remove `$result` variable by inverting the condition and using return statements.

**Comment by Alroniks** [comment]
Line it not changes, so it is a bit out of scope, but why in this line login retrieved from passed login, but identifier - from the current user?
```php
$this->getLockout()->setLockedStatus(UserHelper::getUserLogin($user), $currentUser->getIdentifier());
```

**Review by AndreyNegrov — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**